client/zoos/src/App.js

```diff
@@ -0,0 +1,22 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import './App.css';
+
+export default function App() {
+  const [zoos, setZoos] = useState('');
+
+  useEffect(() => {
+    (async function getData() {
```

Comment:
Excellent suggestion and great implementation.
Glad you were able to utilize this.